test/integration/basic/test/dynamic.js

```diff
@@ -53,23 +53,35 @@ export default (context, render) => {
         }
       })
     })
-    describe('ssr:false option', () => {
-      it('Should render loading on the server side', async () => {
-        const $ = await get$('/dynamic/no-ssr')
-        expect($('body').html()).not.toContain('"dynamicIds"')
-        expect($('p').text()).toBe('loading...')
-      })
-
-      it('should render the component on client side', async () => {
-        let browser
-        try {
-          browser = await webdriver(context.appPort, '/dynamic/no-ssr')
-          await check(() => browser.elementByCss('body').text(), /Hello World 1/)
-        } finally {
-          if (browser) {
-            browser.close()
+    describe('SSR', () => {
+      const testRenderComponentClientSide = (path) => {
+        it('should render the component on client side', async () => {
+          let browser
+          try {
+            browser = await webdriver(context.appPort, path)
+            await check(() => browser.elementByCss('body').text(), /Hello World 1/)
+          } finally {
+            if (browser) {
+              browser.close()
+            }
           }
-        }
+        })
+      }
+      describe('ssr:false option', () => {
+        it('should render loading on the server side', async () => {
+          const $ = await get$('/dynamic/no-ssr')
+          expect($('body').html()).not.toContain('"dynamicIds"')
+          expect($('p').text()).toBe('loading...')
+        })
+        testRenderComponentClientSide('/dynamic/no-ssr')
+      })
+      describe('ssr:true option', () => {
+        it('should render the component on client side', async () => {
+          const $ = await get$('/dynamic/ssr')
```

Comment:
This actually doesn't test explicitly setting `ssr: true`, we already have tests that cover the default option

Reply:
updated
let me know if its fine now, thanks